lang/java/reef-runtime-azbatch/src/main/java/org/apache/reef/runime/azbatch/util/AzureBatchHelper.java

```diff
@@ -16,55 +16,66 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.reef.runime.azbatch.client;
+package org.apache.reef.runime.azbatch.util;
 
 import com.microsoft.azure.batch.BatchClient;
 import com.microsoft.azure.batch.auth.BatchSharedKeyCredentials;
 import com.microsoft.azure.batch.protocol.models.*;
-import org.apache.reef.runime.azbatch.util.AzureBatchFileNames;
 
+import org.apache.reef.runime.azbatch.client.AzureBatchJobSubmissionHandler;
+import org.apache.reef.runime.azbatch.parameters.AzureBatchAccountKey;
+import org.apache.reef.runime.azbatch.parameters.AzureBatchAccountName;
+import org.apache.reef.runime.azbatch.parameters.AzureBatchAccountUri;
+import org.apache.reef.runime.azbatch.parameters.AzureBatchPoolId;
+import org.apache.reef.tang.annotations.Parameter;
+
+import javax.inject.Inject;
 import java.io.IOException;
 import java.net.URI;
 import java.util.Collections;
+import java.util.List;
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
 /**
- * A {@link AzureBatchJobSubmissionHelper} for Azure Batch.
+ * A {@link AzureBatchHelper} for Azure Batch.
  */
-public class AzureBatchJobSubmissionHelper implements AutoCloseable {
+public class AzureBatchHelper implements AutoCloseable {
```

Comment:
We can get rid of AutoCloseable (and the close() method) since the close() method never gets called.

Reply:
done.